src/tools/rbd_mirror/group_replayer/Replayer.cc

```diff
@@ -1797,37 +1797,41 @@ void Replayer<I>::shut_down(Context* on_finish) {
     m_stop_requested = false;
```

Comment:
What is the purpose of clearing `m_stop_requested` here and, more generally, the point of having `m_stop_requested` at all?  After checking where it's set it looks to be duplicating `m_state == STATE_COMPLETE`.